frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/validations/vm-settings-tab-validation.ts

```diff
@@ -101,32 +100,22 @@ export const validateOperatingSystem: VmSettingsValidator = (field) => {
 
 const memoryValidation: VmSettingsValidator = (field, options): ValidationObject => {
   const memValueGB = iGetFieldValue(field);
-  if (!memValueGB) {
+  if (memValueGB == null || memValueGB === '') {
     return null;
   }
   const memValueBytes = memValueGB * 1024 ** 3;
-  const validations = getTemplateValidations(options, VMSettingsField.MEMORY);
+  const validations = getTemplateValidations(options);
   if (validations.length === 0) {
-    return null;
+    validations.push(new TemplateValidations()); // add empty validation for positive integer if it is missing one
```

Comment:
why not exit here, empty validation will end up in line 121 anyway ...  ? what am I missing here ?

Reply:
This just adds default validation for positive integer (not empty one), so there is no chance to insert negative values or 0

Reply:
:+1: don't we have some other mechanism to enforce that ?

Reply:
we do, but just on the UI side in our component. IMO it will not hurt to have it also here